documentation/monitoring.md

```diff
@@ -0,0 +1,51 @@
+# Monitoring and Logging
+
+## Logging
+
+### Application logs
+
+- [Sidekiq](https://teaching-vacancies.service.gov.uk/sidekiq)
+- [Check the Rollbar dashboard](https://rollbar.com/dfe/teacher-vacancies/) to see if any Errors are being logged.
+- [Check the Papertrail event log](https://papertrailapp.com/events) to see in real-time if any Errors are being logged.
+
+### Deployment logs
+
+- [Check the GitHub Deploy workflow](https://github.com/DFE-Digital/teaching-vacancies/actions?query=workflow%3ADeploy) to determine if any recent deployments may have caused the issue.
+
+### Cloudfront
+
+- [Cloudfront log dashboard](https://console.aws.amazon.com/cloudfront/v2/home#/logs/E37BGF9R499RSE). When standard logging is enabled, this goes to a [logging S3 bucket](https://console.aws.amazon.com/s3/buckets/530003481352-tv-cloudfront-logs?region=eu-west-2&prefix=production/&showversions=false) recording logs in gzipped format
+
+## Monitoring, Metrics and Usage
+
+### Grafana, Prometheus, and AlertManager
+
+For several infrastructure components on Gov.UK PaaS, we use [Prometheus Exporters](https://prometheus.io/docs/instrumenting/exporters/) to:
+- fetch statistics from another, non-Prometheus system
+- turn those statistics into Prometheus metrics, using a client library
+- expose a `/metrics` URL, and have that URL display the system metrics
```

Comment:
Could you give an example of a /metrics url? It sounds like it would be at https://teaching-vacancies.service.gov.uk/metrics

Reply:
An excellent point. The convention is `/metrics` at the end of the URL.
In our case, we've taken a sidecar pattern, and exposed it as https://prometheus-teaching-vacancies.london.cloudapps.digital/alerts
I've updated the documentation accordingly, and added details of the 4 apps.